### PR TITLE
fix link in CHANGELOG-1.12.md

### PR DESCRIPTION
**Pull request by Pingan2017**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by Pingan2017** [comment]
ping @monopole  Could you take a look and approve this? Thank u!

**Comment by Pingan2017** [comment]
ping @lavalamp @thockin Can you take a look? Thanks

File: CHANGELOG-1.12.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ filename | sha512 hash
 
 ## Known Issues
 
-- Feature [#566](https://github.com/kubernetes/kubernetes/issues/566) enabling CoreDNS as the default for kube-up deployments was dropped from the release due to a scalability memory resource consumption issue observed. If a cluster operator is considering using CoreDNS on a cluster greater than 2000 nodes, it may be necessary to give more consideration to CoreDNS pod memory resource limits and experimentally measure that memory usage versus cluster resource availability.
+- Feature [#566](https://github.com/kubernetes/features/issues/566) enabling CoreDNS as the default for kube-up deployments was dropped from the release due to a scalability memory resource consumption issue observed. If a cluster operator is considering using CoreDNS on a cluster greater than 2000 nodes, it may be necessary to give more consideration to CoreDNS pod memory resource limits and experimentally measure that memory usage versus cluster resource availability.
```
Comment (nikhita) [inline CHANGELOG-1.12.md]: s/enhancements/features

The features repo was renamed as enhancements recently.

**Comment by nikhita** [comment]
@Pingan2017 Sorry for the delay here! Can you make changes as per the comment? Will try to get it approved after that.

File: CHANGELOG-1.12.md (hunk 1 above)
Comment (Pingan2017) [reply CHANGELOG-1.12.md]: Updated. Thanks!

**Comment by liggitt** [comment]
is that change needed anywhere else in the repo? if so, let's find/replace all at once

**Comment by Pingan2017** [comment]
new pr #71979

**Pull request closed without merging**